components/delete_channel_modal/delete_channel_modal.test.tsx

```diff
@@ -87,7 +87,7 @@ describe('components/delete_channel_modal', () => {
             <DeleteChannelModal {...props}/>
         );
 
-        wrapper.find(Modal).props().onExited?.(document.createElement('div'));
+        wrapper.find(Modal).props().onExited!(document.createElement('div'));
```

Comment:
Fixed the previous TypeScript errors by using a non-null assertion operator which makes sense for this case, but I still have to provide an argument for `onExited`. Otherwise, it will throw a TypeScript error:

"Expected 1 arguments, but got 0.", specifically an HTMLElement. It appears an argument is required:

"(method) TransitionCallbacks.onExited?(node: HTMLElement): any"

Just want to make sure I'm not missing anything here. This PR should hopefully be good to go.

Reply:
Can you pass in `null`? Our implementation just doesn't happen to need any arguments. I'm not sure what the `onExited` method is looking for otherwise. It's probably coming from the `react-bootstrap` type definitions.

Reply:
@devinbinnie I did attempt that but I get a TypeScript error:

"Argument of type 'null' is not assignable to parameter of type 'HTMLElement'."

Reply:
Any other thoughts on this one? I’m out of ideas on how to make TypeScript happy without passing an HTMLElement in. :)

Reply:
Yeah, I guess we'll have to live with that for now.
As long as the comment is there and the tests are passing we should be okay. Thanks for looking into this :)